nemo/utils/sequence_packing_utils.py

```diff
@@ -115,7 +115,7 @@ def create_hist(dataset: np.array, truncate_seq_len: int):
     logging.info("Creating histogram from tokenized dataset...")
 
     sequences = collections.defaultdict(list)
-    counts = [0] * truncate_seq_len
+    counts = [0] * (truncate_seq_len + 1)
```

Comment:
thanks for fixing this, line 132 below should need the same fix

Reply:
Done.